dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/config/ServiceConfig.java

```diff
@@ -71,4 +87,25 @@ public DefaultOutboundMessageBatchService defaultOutboundMessageBatchService(
   public ResourceBundleManager resourceBundleManager() {
     return new DefaultResourceBundleManager();
   }
+
+  @Bean("org.hisp.dhis.expression.ExpressionService")
```

Comment:
I am wondering if this definition is actually unnecessary. Since `SqlBuilder` is now a bean, you can now inject it into DefaultExpressionService as usual with (Lombok) constructor injection.

Reply:
Yes, good catch, removed.